testdata/p4_16_samples/issue3219.p4

```diff
@@ -0,0 +1,9 @@
+bit<4> func1() {
```

Comment:
Can we make this a test that produces an output? We'll see whether Gauntlet needs fixing..

Reply:
Done.